src/training/data.py

```diff
@@ -30,8 +32,106 @@
 from open_clip.tokenizer import HFTokenizer
 
 
+def get_collate_fn(dataset, is_flava=False):
+    if is_flava:
+        assert isinstance(dataset.tokenize, HFTokenizer)
+        hf_tokenizer = dataset.tokenize.tokenizer
+        # The following special tokens are required for
+        # the data collator to work correctly.
+        assert getattr(hf_tokenizer, 'pad_token', False)
+        assert getattr(hf_tokenizer, 'mask_token', False)
+        mlm_collator = DataCollatorForLanguageModeling(
```

Comment:
Hmm, the original FLAVA model uses normal MLM for the text but whole word mask MLM for the multimodal MLM. We should maybe merge that later for both to use normal MLM. (Maybe leave it as the original for now).